proto/multipoolermanagerdata.proto

```diff
@@ -65,25 +65,16 @@ message SetReadOnlyRequest {
 message SetReadOnlyResponse {
 }
 
-// PromoteStandby PostgreSQL standby server to primary
-message PromoteStandbyRequest {
+// PgPromote PostgreSQL standby server to primary
```

Comment:
@deepthi - addressing comments from the previous PR. One follow up question, I left two methods here:
- **PgPromote** that is strictly a wrapper for the postgres call to make a promotion from the postgres perspective.
- **Promote**: More specific to multigres that is the consensus related one. 

Maybe we just need one? And we don't expose PgPromote via RPC?